src/main/java/io/connect/scylladb/ScyllaDbSinkConnectorConfig.java

```diff
@@ -65,14 +64,14 @@
   private static final Pattern TOPIC_KS_TABLE_SETTING_PATTERN =
           Pattern.compile("topic\\.([a-zA-Z0-9._-]+)\\.([^.]+|\"[\"]+\")\\.([^.]+|\"[\"]+\")\\.(mapping|consistencyLevel|ttlSeconds|deletesEnabled)$");
 
-  static final Map<String, ProtocolOptions.Compression> CLIENT_COMPRESSION = 
+  static final Map<String, ProtocolOptions.Compression> CLIENT_COMPRESSION =
```

Comment:
In general, I don't think that formatting changes should go along in the same PR as functional modifications. it makes harder and confusing to review the code.